### ISSUE TITLE
@callback gets called 2x times on initial load

### ISSUE THREAD

**Issue by dobromir-hristov** [NightCatSama/vue-slider-component#37]
Hi. I need to do some ajax requests on change and I tried using the `@callback` but it gets fired 2 times on initial load of the slider. Is this a normal behaviour or something with me is triggering the change?

**Comment by NightCatSama** [comment]
thank you for your issue. this is a bug, and I've been in @2.2.4 version fix it.

**Comment by dobromir-hristov** [comment]
Great job! Just tested ;)

**Comment by dobromir-hristov** [comment]
@NightCatSama Hey I noticed a strange issue. `Callback` event gets called only after I do a click on the line. If I only drag the handles `@callback` never gets called.

**Comment by NightCatSama** [comment]
I drag the handles is calling the `@callback`,  so tell me what is your configuration.
 
this is my conf:
```html
<vue-slider
      v-model="val"
      :lazy="true"
      @callback="cb"
></vue-slider>
```

**Comment by dobromir-hristov** [comment]
Pretty much the same. My `val`  is actually a computed property with a get and set to be able to use it with Vuex but that is not the issue. On first init, on Firefox, if I ONLY drag the handled, I never get a callback emitted. If I click on the line after that drags generate callbacks. I will try to do a fiddle in a few minutes.

**Comment by NightCatSama** [comment]
Oop. Yes, this is a bug, you are very helpful.I will in the next version. 

Thank you.

**Comment by NightCatSama** [comment]
Fix the bug in the version `@2.2.5`.

Thank you again for your time to help me solve this problem.
